### ISSUE TITLE
Add back momentum scrolling of flexbox contents on Mobile Safari

### ISSUE THREAD

**Issue by andrew** [octobox/octobox#721]
It'd be great to get the standard momentum scrolling feel back on mobile safari on both the sidebar and main content.

Article here on how to enable it: https://css-tricks.com/snippets/css/momentum-scrolling-on-ios-overflow-elements/

`overflow` has to be set to `scroll` and we currently use `auto` so we don't get extra scrollbars on desktop, so best place to put something like this in a small-screen media query:

```scss
@include media-breakpoint-down(md) {
  .flex-sidebar, .flex-main {
    overflow-y: scroll;
    -webkit-overflow-scrolling: touch;
  }
}
```

**Comment by samjacobclift** [comment]
PR 👍  https://github.com/octobox/octobox/pull/748

**Comment by andrew** [comment]
Fixed in #748
